tools/ptree-overload/ptree-overload.c

```diff
@@ -93,8 +91,8 @@ usage(void)
         "-k nkeys   Number of keys\n"
         "-p nptombs Number of ptombs\n"
         "-Z config  Path to global config file\n"
-        "-h      Print this help menu\n"
-        , progname);
+        "-h         Print this help menu\n",
+        progname);
```

Comment:
Myabe as well alphabetize the options while you're here, AaBbCcDd...

Reply:
Sure will do.